sphinx/registry.py

```diff
@@ -65,6 +65,9 @@ def __init__(self):
         #: autodoc documenters; a dict of documenter name -> documenter class
         self.documenters = {}           # type: Dict[unicode, Type[Documenter]]
 
+        #: css_files; a list of tuple of filename and attributes
+        self.css_files = []             # type: List[Tuple[unicode, Dict[unicode, unicode]]]
```

Comment:
I think this implementation strongly binds to html related builders.
To be clarity, is the idea that registry has responsibility for setting irrespective of builder?

Reply:
Surely this is used for HTML builders. I know they are bundled, but not core components. So storing them onto registry is a bit strange.
But we already provide `app.add_stylesheet()` as a part of Sphinx API. So I believe this does not change the responsibilities of our modules.